MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/storage/FileSource.java

```diff
@@ -293,12 +293,7 @@ public static void setResourcesCachePath(@NonNull final String path,
     final Context applicationContext = Mapbox.getApplicationContext();
     final FileSource fileSource = FileSource.getInstance(applicationContext);
 
-    if (fileSource.isActivated()) {
```

Comment:
What happens now if we would try to change the cache path while we're processing tiles? What happens if the map is rendering and we call this method, are we going to crash or will we be able to safely reset the database pointer?

Reply:
Not seeing anything wrong during my test. @alexshalamov for visibility.